src/store/state/tombstone-model.ts

```diff
@@ -2,5 +2,6 @@ import { TombStoneIF } from '@/interfaces'
 
 export const tombStoneModel: TombStoneIF = {
   keycloakRoles: [],
-  authRoles: []
+  authRoles: [],
+  entityType: null
```

Comment:
When our typescript settings are set back to full strict, in this case you will get a `strictnullcheck` error for initializing with null. To allow this, you might want to add `| null` in the interface to allow this. Also, might not be a bad idea, during development to enable all the ts strict settings in tsconfig.json file, just to avoid these when we do revert back.

Reply:
Sure